scheduling/round_robin_scheduling_algorithm.py

```diff
@@ -0,0 +1,102 @@
+"""
+Round Robin is a CPU scheduling algorithm.
+In Round Robin each process is assigned a fixed time slot in a cyclic way.
+https://en.wikipedia.org/wiki/Round-robin_scheduling
+"""
+
+
+def calculate_waiting_time(processes, n, burst_time, waiting_time, quantum):
+    """
+    This function calculates the waiting time of some processes that have a
+    specified duration time.
+        Return: The waiting time for each process.
```

Comment:
This function should end with a Python `return` statement that provides the caller the waiting times...

`waiting_times = calculate_waiting_times()`